pkg/checks/latency/latency.go

```diff
@@ -71,7 +71,6 @@ type result struct {
 
 // metrics defines the metric collectors of the latency check
 type metrics struct {
-	duration      *prometheus.GaugeVec
 	totalDuration *prometheus.GaugeVec
 	count         *prometheus.CounterVec
 	histogram     *prometheus.HistogramVec
```

Comment:
Please move the metrics struct to the `pkg/checks/latency/metrics.go` file so everything is always in the same location.

Reply:
Done